src/epgsql_sock.erl

```diff
@@ -189,8 +195,11 @@ set_attr(connect_opts, ConnectOpts, State) ->
 
 %% XXX: be careful!
 -spec set_packet_handler(atom(), pg_sock()) -> pg_sock().
-set_packet_handler(Handler, State) ->
-    State#state{handler = Handler}.
+set_packet_handler(Handler, State0) ->
+    State = State0#state{handler = Handler},
+    Active = get_socket_active(State),
+    setopts(State, [{active, Active}]),
```

Comment:
Don't understand why do we need it here. Isn't the one in `set_net_socket` enough?

But it won't harm here anyway I guess...